middlewares/validators/user.js

```diff
@@ -260,17 +260,18 @@ const validateImageVerificationQuery = async (req, res, next) => {
 };
 
 async function validateUpdateRoles(req, res, next) {
-  const schema = joi.object().strict().min(1).max(1).keys({
+  const schema = joi.object().strict().min(1).max(2).keys({
+    // either member or archived with reason (optional) is allowed
     member: joi.boolean(),
     archived: joi.boolean(),
+    reason: joi.string().optional(), // reason is optional
```

Comment:
why are we making reason optional....cant we pass default as empty string if reason is not present